src/masternode.cpp

```diff
@@ -17,91 +15,35 @@
 
 
 CMasternode::CMasternode() :
-    vin(),
-    addr(),
-    pubKeyCollateralAddress(),
-    pubKeyMasternode(),
-    lastPing(),
-    vchSig(),
-    sigTime(GetAdjustedTime()),
-    nLastDsq(0),
-    nTimeLastChecked(0),
-    nTimeLastPaid(0),
-    nTimeLastWatchdogVote(0),
-    nActiveState(MASTERNODE_ENABLED),
-    nCacheCollateralBlock(0),
-    nBlockLastPaid(0),
-    nProtocolVersion(PROTOCOL_VERSION),
-    nPoSeBanScore(0),
-    nPoSeBanHeight(0),
-    fAllowMixingTx(true),
-    fUnitTest(false)
+    masternode_info_t{ MASTERNODE_ENABLED, PROTOCOL_VERSION, GetAdjustedTime()},
```

Comment:
I don't know what is the best way to do it,  there are different approaches. Current code doesn't contain errors, and we need more new c++ using in our code,  it makes it more simple and readable. But I don't sure about this initializer ctors.)